packages/suite/src/views/wallet/coinmarket/buy/detail/components/WaitingForPayment/index.tsx

```diff
@@ -54,15 +54,15 @@ interface Props {
 const WaitingForPayment = ({ transactionId, trade, account }: Props) => {
     const [isWorking, setIsWorking] = useState(false);
     const { submitRequestForm } = useActions({
-        submitRequestForm: coinmarketBuyActions.submitRequestForm,
+        submitRequestForm: coinmarketCommonActions.submitRequestForm,
     });
 
     const goToPayment = async () => {
         setIsWorking(true);
         const returnUrl = await createTxLink(trade, account);
         invityAPI.getBuyTradeForm({ trade, returnUrl }).then(response => {
             if (response) {
-                submitRequestForm(response);
+                submitRequestForm(response.form);
```

Comment:
nitpick: any reason to mix `async/await` with `then` syntax? (i didn't noticed that earlier)

Reply:
done